applications/jupyter/main.tf

```diff
@@ -81,15 +81,15 @@ module "jupyterhub" {
   gcs_bucket                    = var.gcs_bucket
 
   # IAP Auth parameters
-  add_auth                = var.add_auth
-  brand                   = var.brand
-  support_email           = var.support_email
-  client_id               = var.client_id
-  client_secret           = var.client_secret
-  default_backend_service = var.default_backend_service
```

Comment:
I think calling it Jupyterhub service ingress name of some sort (relating it to the Jupyterhub service) would be clearer, since it will always be the name of the Jupyterhub load balancer service name. WDYT?

Reply:
Discussed offline, the default backed service could be used in other cases.